onnxruntime/python/tools/microbench/matmul.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+from benchmark import benchmark 
+
+
+def create_inputs_outputs(b1, b2, m, k, n, data_type):
+    np.random.seed(0)
+    a = np.random.rand(b1, b2, m, k).astype(data_type)
+    b = np.random.rand(b1, b2, k, n).astype(data_type)
+    c = np.random.rand(b1, b2, m, n).astype(data_type)
+
+    inputs = {"A": a, "B": b}
+    outputs = {"return_val": c}
+ 
+    return inputs, outputs
+
+
+def add_benchmark_case(benchmark_cases, batch_size, seq_len, hidden_size, intermediate_dimension, num_heads, data_type, model):
+    benchmark_cases += [
+        (1, batch_size, seq_len, hidden_size, hidden_size, data_type, model),
+        (1, batch_size, seq_len, intermediate_dimension, hidden_size, data_type, model),
+        (1, batch_size, seq_len, hidden_size, intermediate_dimension, data_type, model),
+        (batch_size, num_heads, seq_len, seq_len, int(hidden_size / num_heads), data_type, model),
+    ]
+
+
+def create_benchmark_cases(fp16=True):
```

Comment:
use precision? in case you will need other precisions later.

Reply:
Nice. Done.